src/regtech_data_validator/phase_validations.py

```diff
@@ -31,6 +31,9 @@
 )
 from regtech_data_validator.checks import SBLCheck, Severity
 
+PHASE_1_TYPE = "Syntactical"
+PHASE_2_TYPE = "Logical"
```

Comment:
These seem like a good candidate for an `enum`, similar to `Severity`. Maybe something like:

```python
class ValidationPhase(StrEnum):
    SYNTACTICAL = "Syntactical"
    LOGICAL = "Logical"
```

...and just abandon the whole "Phase 1/2" bit that doesn't have much meaning outside the internals of this codebase.

Reply:
Thought about making this an enum, shouldn't second guess myself.  Pushed changes.  Moved the enum to the file that uses it.

Reply:
👍 Only odd numbered guesses allowed.